isis/src/base/objs/LineEquation/LineEquation.cpp

```diff
@@ -90,6 +90,7 @@ namespace Isis {
     }
     else if(!p_slopeDefined) {
       p_slope = (p_y[0] - p_y[1]) / (p_x[0] - p_x[1]);
+      p_slopeDefined = true;
```

Comment:
Why is this change here? Is this related to testing?

Reply:
Sorry, forgot about that change.
When writing the tests, I noticed that the flags p_slopeDefined and p_interceptDefined were always false, meaning that the slope and intercept had to be recalculated every time their respective methods were called. This also resulted the methods haveSlope() and haveIntercept() always returning false whether or not the slope and intercept had been defined. I talked with Stuart about it and he said I should just go ahead and change it.